Retail tech review - images and installation (#863)
SUSE/doc-susemanager: modules/administration/pages/image-management.adoc

For more information, see the documentation of the underlying system.
====

[WARNING]
====
The build host must be a Salt client.

> Do not install the build host as a traditionally managed client.

Reviewer: ```suggestion
Do not install the build host as a traditional client.
```
Now, we call these beasts just "traditional client". Or is the longer version also fine, @Loquacity ?

Reviewer 2: I prefer shorter.

====

[[at.images.kiwi.buildhost]]
=== Create a Build Host

To build all kinds of images with {productname}, create and configure a build host.
OS Image build hosts are Salt clients running on {sles}{nbsp}12 (SP3 or later) or {sles}{nbsp}11 SP4.

This procedure will guide you through the initial configuration for a build host.

[IMPORTANT]
====
The operating system on the build host must match the operating system on the targeted image.

For example, build {sles}{nbsp}12 based images on a build host running {sles}{nbsp}12 OS version.
Build {sles}{nbsp}11 based images on a build host running {sles}{nbsp}11 OS version.
====

Configure the build host in the {productname} {webui}:

. Select a client that will be designated as a build host from the menu:Systems[Overview] page.

> . Navigate to menu:System Details[Properties] page, enable the [guimenu]``Add-on System Type:`` [guimenu]``OS Image Build Host`` and confirm with btn:[Update Properties].

Reviewer: ```suggestion
. Navigate to the menu:System Details[Properties] tab, enable the [guimenu]``Add-on System Type`` [guimenu]``OS Image Build Host``.
Confirm with btn:[Update Properties].
```
Even if a label is finished with ":", I'd not add the colon if you reference the name of the label in the running text.

+
image::os-image-build-host.png[scaledwidth=80%]
+

> . Navigate to menu:System Details[Software > Software Channels] page, enable {productname} Client tools (for example: ``SLE-Manager-Tools12-Pool`` and ``SLE-Manager-Tools12-Updates``).

Reviewer: ```suggestion
. Navigate to the menu:System Details[Software > Software Channels] tab, enable {productname} Client tools (for example: ``SLE-Manager-Tools12-Pool`` and ``SLE-Manager-Tools12-Updates``).
```

For more information, see the documentation of the underlying system.
====

[WARNING]
====
The build host must be a Salt client.
Do not install the build host as a traditionally managed client.
====

[[at.images.kiwi.buildhost]]
=== Create a Build Host

To build all kinds of images with {productname}, create and configure a build host.
OS Image build hosts are Salt clients running on {sles}{nbsp}12 (SP3 or later) or {sles}{nbsp}11 SP4.

This procedure will guide you through the initial configuration for a build host.

[IMPORTANT]

> .Relation between image and build host OS version

Reviewer 2: ```suggestion
```

====

> Generally build host OS version should be the running the same OS version as the targeted image.

Reviewer 2: ```suggestion
The operating system on the build host must match the operating system on the targeted image.
```

> - {sles}{nbsp}12 based images should be build on build host running {sles}{nbsp}12 OS version

Reviewer 2: ```suggestion
For example, build {sles}{nbsp}12 based images on a build host running {sles}{nbsp}12 OS version.
```

> - {sles}{nbsp}11 based images should be build on build host running {sles}{nbsp}11 OS version

Reviewer 2: ```suggestion
Build {sles}{nbsp}11 based images on a build host running {sles}{nbsp}11 OS version.
```

====

> From the {productname} {webui}, perform these steps to configure a build host:

Reviewer 2: ```suggestion
Configure the build host in the {productname} {webui}:
```

. Select a client that will be designated as a build host from the menu:Systems[Overview] page.

> . From the menu:System Details[Properties] page, enable the [guimenu]``Add-on System Type:`` [guimenu]``OS Image Build Host`` and confirm with btn:[Update Properties].

Reviewer 2: ```suggestion
. Navigate to menu:System Details[Properties] page, enable the [guimenu]``Add-on System Type:`` [guimenu]``OS Image Build Host`` and confirm with btn:[Update Properties].
```

+
image::os-image-build-host.png[scaledwidth=80%]
+

> . From the menu:System Details[Software > Software Channels] page, enable {productname} Client tools (e.g. ``SLE-Manager-Tools12-Pool`` and ``SLE-Manager-Tools12-Updates``).

Reviewer 2: ```suggestion
. Navigate to menu:System Details[Software > Software Channels] page, enable {productname} Client tools (for example: ``SLE-Manager-Tools12-Pool`` and ``SLE-Manager-Tools12-Updates``).
```
